src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -120,6 +120,22 @@ internal LibraryViewController(Window dynamoView, ICommandExecutive commandExecu
             {
                 WebBrowserUserDataFolder = webBrowserUserDataFolder.FullName;
             }
+
+            /// Create and add the library view to the WPF visual tree.
+            /// Also load the library.html and js files.
+            LibraryViewModel model = new LibraryViewModel();
+            LibraryView view = new LibraryView(model);
+
+            //Adding the LibraryView to the sidebar ensures that the webview2 component is visible.
+            var sidebarGrid = dynamoWindow.FindName("sidebarGrid") as Grid;
+            sidebarGrid.Children.Add(view);
+
+            browser = view.mainGrid.Children.OfType<WebView2>().FirstOrDefault();
+            browser.Loaded += Browser_Loaded;
+            browser.SizeChanged += Browser_SizeChanged;
+
+            LibraryViewController.SetupSearchModelEventsObserver(browser, dynamoViewModel.Model.SearchModel,
+                    this, this.customization);
```

Comment:
moved contents of method AddLibraryView inside  the Constructor. This way we ensure the webview2 controller is visible directly from the constructor.
AddLibraryView was already called immediately after constructor anyway,(https://github.com/DynamoDS/Dynamo/pull/14671/files#diff-abf7d5091e6fead4372c07412928ada4ab2664a9018e07716f311e6980993339L38)